<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cellular Automata</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>476</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="gameLayout"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="settingsLayout">
       <item>
        <layout class="QHBoxLayout" name="controlLayout">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="universeSizeLabel">
         <property name="text">
          <string>Universe Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="universeSizeControl">
         <property name="suffix">
          <string> cells</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="universeModeLabel">
         <property name="text">
          <string>Universe Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="universeModeControl"/>
       </item>
       <item>
        <widget class="QLabel" name="cellModeLabel">
         <property name="text">
          <string>Cell Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cellModeControl">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="generationIntervalLabel">
         <property name="text">
          <string>Evolution interval (msec)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="intervalControl">
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="minimum">
          <number>50</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lifetimeLabel">
         <property name="text">
          <string>Lifetime (evolution steps)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="lifetimeControl">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="fileLayout">
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="colorSelectButton">
         <property name="text">
          <string>Custom Color for Cells</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorRandomButton">
         <property name="text">
          <string>Random Color for Cells</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
